CI is timing out trying to resolve the dependencies

The package was never built, see https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=93595

```
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... 
##[error]The operation was canceled.
##[section]Finishing: Run docker build
```

I'll try to force strict to see if we can resolve this.

--- a/.ci_support/linux_.yaml
+++ b/.ci_support/linux_.yaml
@@ -4,9 +4,3 @@ channel_targets:
 - conda-forge main
 docker_image:
 - condaforge/linux-anvil-comp7
-pin_run_as_build:
-  python:
-    min_pin: x.x
-    max_pin: x.x
-python:
-- '2.7'

--- a/.gitattributes
+++ b/.gitattributes
@@ -17,6 +17,7 @@ bld.bat text eol=crlf
 .gitattributes linguist-generated=true
 .gitignore linguist-generated=true
 .travis.yml linguist-generated=true
+.scripts linguist-generated=true
 LICENSE.txt linguist-generated=true
 README.md linguist-generated=true
 azure-pipelines.yml linguist-generated=true

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,25 +1,24 @@
-{% set name = "jupyter-panel-proxy" %}
 {% set version = "0.1.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: jupyter-panel-proxy
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/j/jupyter-panel-proxy/jupyter-panel-proxy-{{ version }}.tar.gz
   sha256: 4d65091afa2fcd07ed16a3da291137e9955a49fa30664fde72a62bd60382a431
 
 build:
-  noarch: python
   number: 0
+  noarch: python
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
-    - python
+    - python >=3
     - pip
   run:
-    - python
+    - python >=3
     - panel
     - jupyter-server-proxy
 

